### PR TITLE
Redux setup

### PR DESCRIPTION
**Pull request by moskdev**
### Description
Added `redux` and related packages
### Changes
Added `redux` and related packages, created simple test for redux installed, provided close to documentation folder structure

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +33,10 @@
     ]
   },
   "devDependencies": {
-    "gh-pages": "^2.1.1"
+    "gh-pages": "^2.1.1",
+    "react-redux": "^7.1.1",
+    "redux": "^4.0.4",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0"
```
Comment (taras1313) [inline package.json]: except of redux logger all marked deps should be in prod dependencies (redux code goes on prod)

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
+    counter: state.appReducer.counter
+  }
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    actions: {
+      increaseCounter: () => dispatch(increaseCounter()),
+      decreaseCounter: () => dispatch(decreaseCounter())
+    }
```
Comment (taras1313) [inline src/App.js]: to avoid duplications we can use here bindActionCreators

actions: bindActionCreators({increaseCounter, decreaseCounter}, dispatch)

Comment (moskdev) [reply src/App.js]: makes sense, will do better in components that will stay in the application

File: package.json (hunk 1 above)
Comment (moskdev) [reply package.json]: sorry for my inattentiveness, fixed

**Pull request merged**